lib/cli.js

```diff
@@ -2,9 +2,21 @@ const doc = `
 Usage: legofy
 
 Options:
-  -h, --help             Bring help.
-  -v, --version          Show version.
-  -f, --format=<format>  Set output format ('png' or 'jpeg') [default: jpeg].
+  -h, --help                   Bring help.
+  -v, --version                Show version.
+  -f, --format=<format>        Set output format ('png' or 'jpeg')
+                               [default: jpeg].
+  -m, --mode=<mode>            Preset for the performance/quality options
+                               [default: good].
+  --pixel-interval=<interval>  Set pixel interval for average color
+                               computation (higher means faster but less
+                               accurate).
+  --pattern-quality=<quality>  Canvas pattern quality.
+  --filter=<quality>           Canvas filter quality (same values as pattern
+                               quality).
+
+The options taking a quality argument can be one of 'fast', 'good', 'best',
```

Comment:
Well I'm just proxying the option to `canvas`, I don't want to loose configurability.

Though we could just document `fast` and `good` and leave a link.